sds_data_manager/lambda_code/SDSCode/database/models.py

```diff
@@ -27,3 +22,84 @@ class UniversalSpinTable(Base):
     # TODO: create table for repointing and then make
     # a foreign key to universal_spin_table
     repointing_number = Column(Integer, nullable=False)
+
+
+class MetadataTable:
+    """Common Metadata table"""
+
+    id = Column(Integer, primary_key=True)
+    file_name = Column(String, nullable=False)
```

Comment:
Do we have any restrictions on how long the filename can be? If so, you could include that number in the definition here (e.g. `String(100)`)

Reply:
We could so similar thing for other string as well. Like cap them at certain character number since we know they won't be greater than some length.